app/addresses/webapp/manifest.json

```diff
@@ -86,7 +86,7 @@
                     "name": "sap.fe.templates.ListReport",
                     "options": {
                         "settings" : {
-                            "entitySet" : "Addresses",
+                            "contextPath" : "/Addresses",
```

Comment:
Is this change required due to the enabled containment in OData ?

Reply:
Yes, it is also the recommended property by Fiori Elements